pandas/core/indexes/multi.py

```diff
@@ -2676,10 +2676,15 @@ def _partial_tup_index(self, tup, side="left"):
                 return start + section.searchsorted(loc, side=side)
 
             idx = self._get_loc_single_level_index(lev, lab)
-            if k < n - 1:
+            if isinstance(idx, slice) and k < n - 1:
```

Comment:
comment here about why this is necesssary

Reply:
Done

Reply:
ok.  also "GH#37707" pls

Reply:
Done